### ISSUE TITLE
[Bug] Таймаут новой капчи 2ch.hk

### ISSUE THREAD

**Issue by LightGameStudio42** [SthephanShinkufag/Dollchan-Extension-Tools#1501]
Использую последнюю версию скрипта, переустанавливал.

**Опишите вашу ошибку**
Либо после ошибки капчи "Капча сломалась" или подобный текст, в общем, после обновления капчи появляется плашка "Таймаут капчи" и отсчёт на ~300 секунд. При этом открыто несколько вкладок сайта. Без доллчана таймаута не случается. При этом появление таймаута не связано с частотой постинга. И после обновления капчи после таймаута на ней скачет время (200 секунд, а через секунду 70 и обратно) до выбора первого квадратика.
Иногда помогет обновление страницы.

В консоли следующее (не знаю, связано ли с доллчаном):
`error SyntaxError: Unexpected end of JSON input
    at EmojiCaptcha.onEmojiButtonClick (EmojiCaptcha.js:456:108)
/api/captcha/emoji/click:1 
Failed to load resource: the server responded with a status of 404 ()`

**Описание браузера и Dollchan**
1. Chrome 128.0.6613.120 (Official Build)
2. v24.9.16.0.987a9ab.es6
4. Violentmonkey 2.25.0

**Comment by SthephanShinkufag** [comment]
Не знаю, таймаут есть и без куклы.

**Comment by LightGameStudio42** [comment]
> Не знаю, таймаут есть и без куклы.

Он появляется после 10 ответов по дефолту.
С дч же случается, что отхожу от пк на полчаса, по возвращению разворачиваю браузер, обновляю капчу, а там таймаут.

**Comment by rbo23** [comment]
Каптча вообще ужасно работает. Постоянно пишет, что неправильно решил и, да, таймаут этот. При включении/выключении куклы он пропадает

**Comment by LightGameStudio42** [comment]
Иногда после обновления капча вообще отваливается

Что лично мне помогает в случае таймаута:
1. Скорпировать уже написанный в ответ текст, чтобы не потерять его
2. Отключить dollchan
3. Обновить страницу
4. Включить dollchan
5. Обновить страницу
6. Повторять пока таймаут не пропадёт (обычно хватает одной - двух итераций)

![image](https://github.com/user-attachments/assets/d0407e5e-163a-40c5-9e74-e19de3cf4d3b)

**Comment by LightGameStudio42** [comment]
> Каптча вообще ужасно работает. Постоянно пишет, что неправильно решил и, да, таймаут этот. При включении/выключении куклы он пропадает. И это именно что вина куклы

Что помогает - это включать - выключать скрипт ипосле каждого шага жать F5. Главное не забыть скопировать написанный текст чтобы не потерять.

**Comment by xobgoblin** [comment]
Да, также рапортую о имеющейся проблеме. Постоянно вылезает таймаут, неверная капча (при верном выборе иконок), а также имеется заметная задержка при выборе пиктограмм в капче - она отсутствует при выключенной кукле. Приходится постоянно жать F5/выключать куклу. Не то, чтобы это делало куклоскрипт неюзабельным, но неудобство доставляет.

**Comment by LightGameStudio42** [comment]
Иногда вообще перестаёт загружаться, если обновить после т.н. "таймаута" (именно с доллчанов), от лог

```
TypeError: Cannot read properties of null (reading 'appendChild')TypeError: Cannot read properties of null (reading 'appendChild')
    at $alert (board.js?cache=46:6241:20)
    at Object._renderLoaded (board.js?cache=46:2060:13)
    at board.js?cache=46:2216:22
    at complete (board.js?cache=46:1284:17)
    at Object.onsuccess [as success] (board.js?cache=46:1353:25)
    at i (board.js?cache=46:5:27603)
    at Object.fireWith [as resolveWith] (board.js?cache=46:5:28369)
    at A (board.js?cache=46:7:13866)
    at XMLHttpRequest.<anonymous> (board.js?cache=46:7:16154)
```

**Comment by LightGameStudio42** [comment]
Вообще что после ручного обновления по кнопке, что при сбросе после "слома", что после "таймаута" первые **ровно два** ввода капчи всегда "некорректные".

**Comment by LightGameStudio42** [comment]
Можно добавить рядом с каждым сложением кнопку сохранения.
Юзкейс: прикладываешь 4 скриншота, капча пишет таймаут. Вместо того, чтобы ждать 5 минут и 2 неудачных попытки, можно просто скачать каждую картинку и после обновления страницы приложить заново.

**Comment by LightGameStudio42** [comment]
Капча ломается даже при открытии темы

**Comment by LightGameStudio42** [comment]
+ после обновления страницы слетают галки в чекбоксах в форме отправления

**Comment by LightGameStudio42** [comment]
Не зугражется капча у новых тредов вообще

`TypeError: Cannot read properties of null (reading 'appendChild')TypeError: Cannot read properties of null (reading 'appendChild')
    at $alert (board.js?cache=46:6241:20)
    at Object._renderLoaded (board.js?cache=46:2060:13)
    at board.js?cache=46:2216:22
    at complete (board.js?cache=46:1284:17)
    at Object.onsuccess [as success] (board.js?cache=46:1353:25)
    at i (board.js?cache=46:5:27603)
    at Object.fireWith [as resolveWith] (board.js?cache=46:5:28369)
    at A (board.js?cache=46:7:13866)
    at XMLHttpRequest.<anonymous> (board.js?cache=46:7:16154)`
